src/main/java/uk/gov/companieshouse/controller/DissolutionController.java

```diff
@@ -0,0 +1,61 @@
+package uk.gov.companieshouse.controller;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import uk.gov.companieshouse.model.dto.CreateDissolutionRequestDTO;
+import uk.gov.companieshouse.model.dto.CreateDissolutionResponseDTO;
+import uk.gov.companieshouse.service.DissolutionService;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.validation.Valid;
+
+@RestController
+@RequestMapping("/dissolution-request/{company-number}")
+public class DissolutionController extends BaseController {
+
+    private final DissolutionService dissolutionService;
+
+    public DissolutionController(DissolutionService dissolutionService) {
+        super();
+        this.dissolutionService = dissolutionService;
+    }
+
+    @Operation(summary = "Create Dissolution Request", tags = "Dissolution")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "201", description = "Dissolution Request created"),
+            @ApiResponse(responseCode = "400", description = "Missing headers"),
+            @ApiResponse(responseCode = "401", description = "Unauthorised"),
+            @ApiResponse(responseCode = "409", description = "Dissolution Request already exists for company"),
+            @ApiResponse(responseCode = "422", description = "Invalid request format"),
+            @ApiResponse(responseCode = "500", description = "Internal server error")
+    })
+    @PostMapping(consumes = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<CreateDissolutionResponseDTO> submitDissolutionRequest(
+        @RequestHeader("ERIC-identity") String userId,
+        @RequestHeader("ERIC-Authorised-User") String email, // TODO - extract email from header
```

Comment:
TODO - this header is a comma separated string, not just the email. Need to confirm actual format so that I can test it.